constants.cs

```diff
@@ -89,7 +89,8 @@ public enum TYPES {
 public enum KINDS {
     VAR       =     TOKENS.VAR,
     PROCEDURE =     TOKENS.PROCEDURE,
-    FUNCTION  =     TOKENS.FUNCTION
+    FUNCTION  =     TOKENS.FUNCTION,
+    PARAMETER
```

Comment:
This is implicitly set to '13' as it follows function in the TOKENS enum. It seems risky to just let it do that since it could cause hard to find issues later on. While it doesn't matter what this is, just set it to something outside of our TOKENS enum like 999.